src/main/java/com/minecolonies/coremod/colony/requestsystem/management/IStandardRequestManager.java

```diff
@@ -21,22 +21,22 @@
 public interface IStandardRequestManager extends IRequestManager
 {
     @NotNull
-    BiMap<IToken, IRequestResolverProvider> getProviderBiMap();
+    BiMap<IToken<?>, IRequestResolverProvider> getProviderBiMap();
 
     @NotNull
-    BiMap<IToken, IRequestResolver> getResolverBiMap();
+    BiMap<IToken<?>, IRequestResolver> getResolverBiMap();
 
     @NotNull
-    BiMap<IToken, IRequest> getRequestBiMap();
+    BiMap<IToken<?>, IRequest> getRequestBiMap();
 
     @NotNull
-    Map<IToken, ImmutableCollection<IToken>> getProviderResolverMap();
+    Map<IToken<?>, ImmutableCollection<IToken<?>>> getProviderResolverMap();
 
     @NotNull
-    Map<IToken, Set<IToken>> getResolverRequestMap();
+    Map<IToken<?>, Set<IToken<?>>> getResolverRequestMap();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)